airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -541,7 +541,7 @@ def test_additional_properties_is_true(self, actual_connector_spec: ConnectorSpe
         )
         if additional_properties_values:
             assert all(
-                [additional_properties_value is True for additional_properties_value in additional_properties_values]
+                [additional_properties_value is not False for additional_properties_value in additional_properties_values]
```

Comment:
the file based spec includes "additionalProperties" fields because the formats aren't part of the spec. we can make the tests pass by checking it's not set to False OR somehow enforce the format fields in the spec (maybe using generics)